torch/_refs/nn/functional/__init__.py

```diff
@@ -262,6 +263,37 @@ def relu(a: TensorLikeType, inplace: bool = False) -> TensorLikeType:
     return torch.where(torch.le(a, 0), 0, a)
 
 
+@register_decomposition(aten.channel_shuffle)
+@aten.channel_shuffle.default.py_impl(DispatchKey.CompositeImplicitAutograd)
+@aten.channel_shuffle.default.py_impl(DispatchKey.Autograd)
```

Comment:
If this function is its own backward, I don't think we should bother with using `py_impl` as we'll get the backward for free anyway.